### PR TITLE
Update homematicip_cloud.markdown

### PR DESCRIPTION
**Pull request by steckenpferd**
HmIP Cloud Documentation update for 0.87.0

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `current` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by steckenpferd** [comment]
Please add label current

File: source/_components/homematicip_cloud.markdown [hunk 1]
```diff
@@ -11,6 +11,7 @@ logo: homematicip_cloud.png
 ha_category: Hub
 ha_iot_class: "Cloud Push"
 ha_release: 0.66
+last update: 0.87.0
```
Comment (klaasnicolaas) [inline source/_components/homematicip_cloud.markdown]: this option did not exist, so please omit it.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/homematicip_cloud.markdown (hunk 1 above)
Comment (steckenpferd) [reply source/_components/homematicip_cloud.markdown]: Ok - Done

**Comment by steckenpferd** [comment]
When could it be added?

**Review by klaasnicolaas — APPROVED** [review]

**Comment by frenck** [comment]
@steckenpferd Thank you for your PR, that is appreciated. As you might have noticed, we have quite a few PR's that needs our attention. Since we are all volunteering our time to HA, it sometimes can take a little longer, please have some patience.

Thank you for your understanding.

**Pull request merged**